feat(uipath-agents): integrate coded agents in flow (#242)
UiPath/skills: skills/uipath-planner/SKILL.md

Reviewer: Those might create a confusion that `uipath-mestro-flow` handles solution deployment, rather than `uipath-platform` which leverages the `uip solution` tool.

Rename the flag to `Deploy` and let the other skill internally to handle it.

- The plan contains **no generation skill** (pure `uipath-interact` interaction, pure `uipath-platform` ops, pure read-only diagnostics) → record `Test coverage: N/A`.
- The request is a small modification to an existing automation and the user has not asked for new tests — default to `standard` for touched paths and note the assumption in Decisions & Trade-offs.

### Question 5: Solution scope

Use `AskUserQuestion`:

- **Question:** `Where should this solution live?`
- **Options:**
  - `SW solution` — Build and iterate in Studio Web.
  - `Local solution` — Build and iterate locally in VSCode.

>   - `Something else` — Free-form input.

Reviewer: ```suggestion
- **Question:** `Where should the generated resources be deployed?`
- **Options:**
  - `Yes`
  - `No`
  - `Something else` 
```

> Record the answer in the plan header as `Solution scope: SW | local`. Specialist skills read this field at runtime to decide whether to publish at the end.

Reviewer: ```suggestion
Record the answer in the plan header as `Deploy: Yes | No`. Specialist skills read this field at runtime to decide whether to publish at the end.
```

**Skip this question** when:

> - The user's request already states the intent (e.g., "upload to Studio Web", "keep it local", "just build it").

Reviewer: ```suggestion
- The user's request already states the intent (e.g., "upload", "keep it local", "just build it").
```

**UI targeting:** <agent-builds-you-review / user-indicates / N/A>
**UI capture:** <live-capture / indication-only / N/A>
**Test coverage:** <standard / happy-path / N/A>

> **Solution scope:** <SW | local | N/A>

Reviewer: ```suggestion
**Deploy:** <Yes | No | N/A>
```
